### ISSUE TITLE
Project overview needs a h1 element for a11y

### ISSUE THREAD

**Issue by phillimorland** [liqd/a4-meinberlin#3566]
all pages should have a main header (h1) for a11y so screen readers have something to identify the page. The project overview page doesn't have a title (h1) and I'm not sure where we should try to add one @mcastro-lqd is there anywhere that we could add a title to this page? 
url: https://meinberlin-dev.liqd.net/projekte/
